src/components/Icon/Icon.jsx

```diff
@@ -822,6 +822,25 @@ const icons = (type, backgroundColor, iconColor) => {
           </g>
         </g>
       </g>
+    ),
+    rfp: (
```

Comment:
should we call it `rfp`? The icon can still be used outside of RFP's context in the future, right? I think we should go with a more declarative naming e.g. `horizontalLink`.

Reply:
You are totally right mate, forgot to change it.